services/web/server/src/simcore_service_webserver/publication_handlers.py

```diff
@@ -0,0 +1,56 @@
+import json
+import os
+
+from aiohttp import MultipartReader, hdrs, web
+
+from .login.decorators import RQT_USEREMAIL_KEY, login_required
+
+support_email_address = 'support@osparc.io'
+email_template_name = 'service_submission.html'
```

Comment:
create a configuration for these things. especially the email address could change depending on the platform.

Reply:
call you later if you have time